quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -48,7 +49,7 @@ public ConfigValue getOrDefault(String name, ConfigSourceInterceptorContext cont
     PropertyMapper(String from, String to, String defaultValue, BiFunction<String, ConfigSourceInterceptorContext, String> mapper,
             String mapFrom, boolean buildTime, String description, String paramLabel, boolean mask, Iterable<String> expectedValues, ConfigCategory category) {
         this.from = MicroProfileConfigProvider.NS_KEYCLOAK_PREFIX + from;
-        this.to = to;
+        this.to = to == null ? this.from : to;
```

Comment:
is there a property where to / the quarkus property is actually null? Could you elaborate a bit why we need this?

Reply:
In the next round of changes. These are properties that do not map to other properties but are used as a dependency to others.

Reply:
ah ok